Arm/Memory/Attr.lean

```diff
@@ -0,0 +1,9 @@
+import Lean
+
+open Lean
+
+/-- Provides tracing for the `simp_mem` tactic. -/
+initialize Lean.registerTraceClass `simp_mem
+
+/-- Provides extremely verbose tracing for the `simp_mem` tactic. -/
+initialize Lean.registerTraceClass `simp_mem.info
```

Comment:
I *think* you can write this in a single block, as
```
initialize
  /-- bla -/
  Lean.registerTraceClass ...
  /-- bla -/
  Lean.registerTraceClass ...
```

Matter of taste which is better, feel free to ignore

Reply:
I feel I prefer this style, 'cause it seems easier to move around.